.gitignore

```diff
@@ -50,6 +50,12 @@ gcloud_prod.json
 /cover/
 /node_modules/
 .vscode/
+.google.secret.json
+.prod.env
+.local.env
+.staging.env
 
 # rustler nif binaries
-priv/native/
\ No newline at end of file
+/priv/native
+/native/sqlparser_ex/target
+/native/sqlparser_ex/.config
```

Comment:
don't need the last two under `/native/*`, see above.

Reply:
what do you mean? and this is for local development purposes

Reply:
got it will remove, I did cargo build inside `/native/sqlparser_ex/` hence creating things there and ignoring it but will change it